tests/js/spec/components/group/externalIssueForm.spec.jsx

```diff
@@ -6,13 +6,19 @@ import {SentryAppExternalIssueForm} from 'app/components/group/externalIssueForm
 describe('SentryAppExternalIssueForm', () => {
   let wrapper;
   let group;
+  let sentryApp;
   let sentryAppInstallation;
   let component;
 
   beforeEach(() => {
-    group = TestStubs.Group();
+    group = TestStubs.Group({
+      title: 'ApiError: Broken',
+      shortId: 'SEN123',
+      permalink: 'https://sentry.io/organizations/sentry/issues/123/?project=1',
+    });
     component = TestStubs.SentryAppComponent();
-    sentryAppInstallation = TestStubs.SentryAppInstallation();
+    sentryApp = TestStubs.SentryApp();
+    sentryAppInstallation = TestStubs.SentryAppInstallation({sentryApp});
```

Comment:
need to add `sentryApp` cause of https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils/fetchSentryAppInstallations.jsx#L23